data/munin/groonga_disk_

```diff
@@ -157,17 +157,16 @@ def autoconf
 end
 
 def suggest
-  exist_p = lambda do |variable_name|
-    database_path = ENV[variable_name]
-    database_path and File.exist?(database_path)
+  exist_p = lambda do |database_path_name, pid_file_path_name|
+    File.exist?(ENV[database_path_name]) and File.exist?(ENV[pid_file_path_name])
```

Comment:
Ah, this will raise an error when it's not specified:

```console
$ ruby -e 'File.exist?(nil)'
-e:1:in `exist?': no implicit conversion of nil into String (TypeError)
	from -e:1:in `<main>'
```